framework/src/mysql.js

```diff
@@ -84,12 +84,13 @@ const cast = (val, type) => {
 	if (type === 'string') return String(val);
 	if (type === 'boolean') return Boolean(val);
 	if (type === 'bigInteger') return BigInt(val);
+	if (type === 'json') return JSON.stringify(val);
 	return val;
 };
 
 const resolveQueryParams = params => {
 	const queryParams = Object.keys(params)
-		.filter(key => !['sort', 'limit', 'propBetweens', 'orWhere', 'orWhereWith', 'offset', 'whereIn', 'orWhereIn', 'search', 'aggregate']
+		.filter(key => !['sort', 'limit', 'propBetweens', 'orWhere', 'orWhereWith', 'offset', 'whereIn', 'orWhereIn', 'search', 'aggregate', 'whereJsonSupersetOf']
 			.includes(key))
```

Comment:
Please consider refactoring the code to somethinngn like this.
```suggestion
		.filter(key => !KNOWN_QUERY_PARAMS.includes(key))
```